mandatory/1-fix-functions.js

```diff
@@ -10,20 +10,20 @@
 
 */
 
-function mood() {
-  let isHappy = true;
+function mood(bool) {
+  let isHappy = true; 
 
-  if (isHappy) {
+  if (isHappy==bool) {
```

Comment:
line 14 is redundant. You could evaluate directly the argument parameter. For example:
```
  if (bool) {
  return "I am happy";
  } else {
    return "I am not happy";
  }
  ```

Reply:
yes, fixed it